src/Robo/Commands/Generate/AliasesCommand.php

```diff
@@ -247,7 +247,7 @@ protected function getSiteAliases($site) {
       $remoteHost = $ssh_split[1];
       $remoteUser = $ssh_split[0];
 
-      if ($hosting == 'ace') {
+      if ($hosting == 'ace' || $hosting == 'acp') {
```

Comment:
Instead of limiting this to only ACE or ACP, generate a standard drush alias for every valid hosting type returned except for ACSF - the easiest way to do this is probably `!='acsf'`. Since valid hosting types that could be returned currently are 'free', 'acp', 'ace', 'acsf', 'network', and 'search' and will presumably change as Cloud API v2 gets closer to GA, this will allow for a bit more flexibility

Reply:
Thank you @lcatlett ! I've made the adjustment. I was tempted to add an `else` to the next condition block, but I think leaving it more verbose is better in this case. Let me know if you disagree and I'll gladly update it.

Reply:
@lcatlett Is this change acceptable?